### PR TITLE
implement mutasql approach

### PR DESCRIPTION
**Pull request by Illyrix**

### CODE REVIEW

File: pkg/sqlancer/mutasql.go [hunk 1]
```diff
@@ -0,0 +1,516 @@
...
+	tables := t.GetAllTables()
+	originTableNames := make([]types.CIStr, 0)
+	for _, i := range tables {
+		originTableNames = append(originTableNames, i.Name)
+	}
+	tableNames := m.randTableNames(originTableNames)
+
+	// renamedCase := t.Clone()
+	renamedCase := t
+	renamedCase.ReplaceTableName(tableNames)
```
Comment (mahjonp) [inline pkg/sqlancer/mutasql.go:258]: Why do we need to apply `ReplaceTableName` when  applyTestCase?

**Comment by mahjonp** [comment]
Almost LGTM. 
1) But we can add the new test case(Q', D) back to the (Q, D) collection.
2) We can also mutate  D -> D'(eg. Adding Indexes) when mutating the original case.

File: pkg/sqlancer/mutasql.go (hunk 1 above)
Comment (mahjonp) [reply pkg/sqlancer/mutasql.go:258]: I have got it.

File: cmd/mutasql/main.go [hunk 2]
```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"time"
+
+	"github.com/chaos-mesh/go-sqlancer/pkg/sqlancer"
+	_ "github.com/go-sql-driver/mysql"
```
Comment (Hexilee) [inline cmd/mutasql/main.go:10]: Thrid-party packages and local packages should be imported as different groups.

File: cmd/mutasql/main.go [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+	if err := conf.SetDSN(*dsn); err != nil {
+		panic(err)
+	}
+	conf.ViewCount = *viewCount
+	conf.Depth = *depth
+	conf.Silent = *silent
+	conf.LogLevel = *logLevel
+	conf.EnableHint = *hint
+	conf.EnableExprIndex = *exprIdx
+	// if len(*mode) == 0 {
```
Comment (Hexilee) [inline cmd/mutasql/main.go:49]: Unrelevant comments should be removed?

**Pull request merged**